### ISSUE TITLE
Better regex for raw.githubusercontent.com

### ISSUE THREAD

**Issue by twardoch** [simov/markdown-viewer#22]
With private Github repos, when I click on the `Raw` button for any Markdown document, the URL that is formed looks like this: 
``` 
https://raw.githubusercontent.com/user/repo/master/README.md?token=xxxxxxxxxxxx
```
I’ve modified the regex so that it catches the `?` after the `.md` extension. This works well: 
```
\.(?:markdown|mdown|mkdn|md|mkd|mdwn|mdtxt|mdtext|text)(?:#.*|\?.*)?$
```
Perhaps you’d be willing to improve the defaults in the extension? It’s fantastic anyway!

**Comment by simov** [comment]
Good catch! I'll probably add it.

**Comment by simov** [comment]
The fix is published in version 3.2
